lib/rust/ensogl/component/text/embedded-fonts/build.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this an empty file?

Reply:
This seems to be a rebase issue that I missed. Needs to be removed.

Reply:
Just removed it in my branch.